.changeset/fast-jobs-type.md

```diff
@@ -0,0 +1,6 @@
+---
+"@khanacademy/perseus": major
+"@khanacademy/perseus-editor": major
```

Comment:
I'm open to debating the `major` here. From my perspective: we're changing external-facing types that have a fairly high chance of requiring us to make updates in Webapp. 🤷